scripts/database/pg/migrations/021.do.force-private-true.sql

```diff
@@ -0,0 +1,7 @@
+DO $$
+    BEGIN
+        BEGIN
+          UPDATE cd_profiles SET private = true WHERE private IS NULL;
```

Comment:
Should also alter table to make default true for all new entries? I know it's done in the µs, but doing it in the DB schema seems a little cleaner IMO.